src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetCalculator.ts

```diff
@@ -72,9 +72,9 @@ export class VexFlowMusicSheetCalculator extends MusicSheetCalculator {
     // prepare Vexflow font (doesn't affect Vexflow 1.x). It seems like this has to be done here for now, otherwise it's too slow for the generateImages script.
     //   (first image will have the non-updated font, in this case the Vexflow default Bravura, while we want Gonville here)
     if (this.rules.DefaultVexFlowNoteFont?.toLowerCase() === "gonville") {
-      (Vex.Flow as any).DEFAULT_FONT_STACK = [(Vex.Flow as any).Fonts?.Gonville, (Vex.Flow as any).Fonts?.Bravura, (Vex.Flow as any).Fonts?.Custom];
+      VF.Flow.setMusicFont("Gonville", "Custom");
     } else if (this.rules.DefaultVexFlowNoteFont?.toLowerCase() === "petaluma") {
-      (Vex.Flow as any).DEFAULT_FONT_STACK = [(Vex.Flow as any).Fonts?.Petaluma, (Vex.Flow as any).Fonts?.Gonville, (Vex.Flow as any).Fonts?.Bravura];
+      VF.Flow.setMusicFont("Gonville", "Custom");
```

Comment:
@rvilarl This should be Petaluma, right?

Reply:
No, you wanted to support Gonville only. This would increase the size of the build which is 1094366.

Reply:
Well, for now, for the default build. But once we make a build with all fonts or with only petaluma maybe, this will be supported then, and until then this doesn't hurt, does it? Nobody will try to use Petaluma in a Gonville-only OSMD build.